meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -0,0 +1,41 @@
+DESCRIPTION = "Mender program for remote terminal access."
+HOMEPAGE = "https://mender.io"
+
+SRC_URI = "git://github.com/mendersoftware/mender-shell.git;protocol=https;branch=master"
+
+# See: https://www.yoctoproject.org/docs/2.5.1/dev-manual/dev-manual.html#automatically-incrementing-a-binary-package-revision-number
+SRCREV = "${AUTOREV}"
+PV = "1.0+git${SRCPV}"
```

Comment:
While in development, maybe it should be "0.1" instead of "1.0"?

Reply:
Makes sense to me